src/Utilities.gs

```diff
@@ -39,24 +39,12 @@ function buildUrl_(url, params) {
 function validate_(params) {
   Object.keys(params).forEach(function(name) {
     var value = params[name];
-    if (isEmpty_(value)) {
+    if (!value) {
```

Comment:
Not functionally equivalent to isEmpty_(), but I don't think any of my validations rely on detecting empty arrays or objects.

Reply:
Good point.